src/MovieGrid.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import MovieCard from './MovieCard.js';
+import './MovieGrid.css';
+
+class MovieGrid extends Component {
+  constructor() {
+    super();
+    this.state = {
+      movies: [
```

Comment:
Awesome job here to get us set up with an initial array of movies. Now we just have to get our fetch request hooked up which should display really well with the initial structure that you've created.